site/docs-md/guides/ionic-framework-app.md

```diff
@@ -218,7 +218,7 @@ Next, for the Camera plugin to work, we must configure the "Privacy - Camera Usa
 
 Each setting in `Info.plist` has a low-level parameter name and a high-level name. By default, the property list editor shows the high-level names, but it's often useful to switch to showing the raw, low-level names. To do this, right-click anywhere in the property list editor and toggle "Show Raw Keys/Values."
 
-Locate the `NSCameraUsageDescription` Key (or add it) and set the Value to something that describes why the app needs to use the camera, such as "To Take Photos." The Value will be displayed to the app user when the permission prompt opens.
+Locate the `NSCameraUsageDescription` Key (or add it) and set the Value to something that describes why the app needs to use the camera, such as "To Take Photos." The Value will be displayed to the app user when the permission prompt opens. Do the same for `NSPhotoLibraryAddUsageDescription` Key and `NSPhotoLibraryUsageDescription` Key.
```

Comment:
The sample apps doesn't read or save to the gallery, so those two are not needed for the guide.